integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -414,27 +406,27 @@ private static void runConsumerInOrderToCreateGroup(KroxyliciousTester tester, S
         }
     }
 
-    private static void verifyConsumerGroupsWithList(Admin admin, Set<String> expectedGroupIds) throws Exception {
+    private void verifyConsumerGroupsWithList(Admin admin, Set<String> expectedGroupIds) throws Exception {
         var groups = admin.listConsumerGroups().all().get().stream().map(ConsumerGroupListing::groupId).toList();
         assertThat(groups).containsExactlyInAnyOrderElementsOf(expectedGroupIds);
     }
 
-    private static void verifyTransactionsWithList(Admin admin, Set<String> expectedTransactionalIds) throws Exception {
+    private void verifyTransactionsWithList(Admin admin, Set<String> expectedTransactionalIds) throws Exception {
         var transactionalIds = admin.listTransactions().all().get().stream().map(TransactionListing::transactionalId).toList();
         assertThat(transactionalIds).containsExactlyInAnyOrderElementsOf(expectedTransactionalIds);
     }
 
-    private static void verifyTenant(Admin admin, String... expectedTopics) throws Exception {
+    private void verifyTenant(Admin admin, String... expectedTopics) throws Exception {
         var listTopicsResult = admin.listTopics();
 
         var topicListMap = listTopicsResult.namesToListings().get();
-        assertEquals(expectedTopics.length, topicListMap.size());
+        assertThat(expectedTopics.length).isEqualTo(topicListMap.size());
```

Comment:
```suggestion
        assertThat(expectedTopics).hasSize(topicListMap.size());
```
or

```suggestion
        assertThat(expectedTopics).hasSameSizeAs(topicListMap);
```

Reply:
This doesn't work because `expectedTopics` is an array, not a List, and therefore does not have a size property. Varargs are an implicit conversion to array, so I would have to convert `expectedTopics` to List to make this assertion. I tried both versions, and the one I went with (`assertThat(expectedTopics.length).isEqualTo(topicListMap.size());`) was the less verbose of the two.

Reply:
Sorry! I thought I had checked it worked before suggesting....

AHH so I checked it compiled, but it fails at run time as `topicListMap` is, obviously, a map.

Reply:
Sorry, yes, you're right. It's not failing because `expectedTopics` is an array, it's failing because we're comparing it to a map and not another array. Replacing this with `assertThat(expectedTopics).hasSize(topicListMap.size());` makes a bit more sense, so I've done that.